draft-ietf-wimse-s2s-protocol.md

```diff
@@ -484,6 +484,32 @@ A signed response would be:
 ~~~
 {: title="Signed Response"}
 
+## Comparing the DPoP Inspired Option with Message Signatures
+
+<cref>
+This section is temporarily part of the draft, while we expect the working group
+to select one of these options.
+</cref>
+
+The two options for protecting the workload's traffic vary with respect to implementation
+complexity, extensibility and security. Here is a summary of the main differences between
+{{dpop-esque-auth}} and {{http-sig-auth}}.
+
+- The DPoP-inspired solution is less HTTP-specific, making it easier to adapt for other protocols beyond HTTP. This flexibility is particularly valuable for asynchronous communication scenarios, such as event-driven systems.
+
+-    Message Signatures, on the other hand, benefit from an existing RFC with established implementations. This existing groundwork means that this option could be simpler to deploy.
+
+-    Given that the WIT (Web Interaction Token) is a type of JWT, the DPoP-inspired approach is less complex and technology-intensive than Message Signatures. In contrast, Message Signatures introduce additional layers of technology, potentially increasing the complexity of the overall system.
+
+-    Message Signatures offer superior integrity protection, particularly by mitigating message modification by middleboxes.
+
+-    A key advantage of Message Signatures is that they support response signing.
```

Comment:
Some of the indentation and line breaks are off. I believe it is best to align this.